addons/base/views.py

```diff
@@ -837,8 +838,14 @@ def addon_view_or_download_file(auth, path, provider, **kwargs):
     if hasattr(target, 'get_addon'):
 
         node_addon = target.get_addon(provider)
-
-        if not isinstance(node_addon, BaseStorageAddon):
+        if flag_is_active(request, features.ENABLE_GV):
```

Comment:
Does this maybe also need to check to see if it's osfsftorage, or does that not matter here?

Reply:
I don't think it should matter. As far as I can tell, this view is handling creation of guid for a file on an addon when user clicks to view it for the first time, among some other things.